common/tools/eslint-plugin-azure-sdk/src/rules/ts-versioning-semver.ts

```diff
@@ -64,42 +64,44 @@ export = {
             if (secondPart === undefined) {
               return;
             }
-            const devOrPreviewVer = secondPart.match(/^(dev|preview)(.*)/);
-            if (devOrPreviewVer === null) {
+            const ver = secondPart.match(/^(dev|preview|alpha|beta)(.*)/);
+            if (ver === null) {
               context.report({
                 node: nodeValue,
                 message: `unrecognized version syntax: ${secondPart}`
               });
               return;
             }
 
-            const devOrPreviewVerKeyword = devOrPreviewVer[1];
-            const devOrPreviewVerNumber = devOrPreviewVer[2];
-            switch(devOrPreviewVerKeyword) {
-              case 'preview':
-                if (!/^\.(:?0|(?:[1-9]\d*))$/.test(devOrPreviewVerNumber)) {
+            const verKeyword = ver[1];
+            const verNumber = ver[2];
+            switch (verKeyword) {
+              case "preview":
+              case "alpha":
```

Comment:
For alpha the version will still be a date similar to dev so I don't think this validation is correct.

Reply:
Thanks. I opened: https://github.com/Azure/azure-sdk-for-js/pull/10998.